onnxruntime/core/optimizer/conv_bn_fusion.cc

```diff
@@ -9,184 +9,163 @@ using namespace ONNX_NAMESPACE;
 using namespace ::onnxruntime::common;
 namespace onnxruntime {
 
-Status ConvBNFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level) const {
-  std::vector<onnxruntime::NodeIndex> removed_nodes;
-  for (auto& node : graph.Nodes()) {
-    ORT_RETURN_IF_ERROR(Recurse(node, modified, graph_level));
-
-    if (!graph_utils::IsSupportedOptypeVersionAndDomain(node, "Conv", 1) || 
-        !graph_utils::IsSupportedProvider(node, GetCompatibleExecutionProviders()) ||
-        node.GetOutputEdgesCount() != 1) {
-      continue;
-    }
-
-    const Node& next_node = *node.OutputNodesBegin();
-    if (!graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "BatchNormalization", 7) ||
-        next_node.GetInputEdgesCount() != 1 ||
-        graph.IsNodeOutputsInGraphOutputs(next_node) ||
-        next_node.GetExecutionProviderType() != node.GetExecutionProviderType()) {
-      continue;
-    }
-
-    auto& conv_node = node;
-    const Node& bn_node = next_node;
+Status ConvBNFusion::Apply(Graph& graph, Node& node, bool& modified, bool& /*removed*/) {
+  auto& conv_node = node;
+  const Node& bn_node = *conv_node.OutputNodesBegin();
+
+  // Get value of attribute group
+  const onnxruntime::NodeAttributes& conv_attributes = conv_node.GetAttributes();
+  const ONNX_NAMESPACE::AttributeProto* group_attr = &(conv_attributes.find("group")->second);
+  if (group_attr != nullptr &&
+      group_attr->type() == AttributeProto_AttributeType_INT &&
+      group_attr->has_i() && group_attr->i() != 1) {
+    return Status::OK();
```

Comment:
Comment explaining what the intention of this test is would be good

Reply:
I chatted with @tracysh about this -- it seems this check is not needed, so I removed it.